migrations/0004_create_orders.up.sql

```diff
@@ -0,0 +1,21 @@
+create table orders (
+  id uuid primary key not null default uuid_generate_v4(),
+	created_at timestamp with time zone not null default current_timestamp,
+	updated_at timestamp with time zone not null default current_timestamp,
+	total_price text,
+	merchant_id text,
+	status text
+);
+
+create table order_items (
+  id uuid primary key not null default uuid_generate_v4(),
+  order_id uuid,
```

Comment:
what are your thoughts on using a foreign key constraint here?

Reply:
I like it!
https://github.com/brave-intl/bat-go/pull/226/commits/bea43555003e75c8558bb5ace641ccfc237700f8